ee/clickhouse/queries/funnels/test/__snapshots__/test_funnel.ambr

```diff
@@ -0,0 +1,76 @@
+                                                     steps = 2) step_people_2,
+                                           avg(step_1_average_conversion_time_inner) step_1_average_conversion_time,
+                                           median(step_1_median_conversion_time_inner) step_1_median_conversion_time
+  FROM
+    (SELECT person_id,
+            steps,
+            avg(step_1_conversion_time) step_1_average_conversion_time_inner,
+            median(step_1_conversion_time) step_1_median_conversion_time_inner
+     FROM
+       (SELECT person_id,
+               steps,
+               max(steps) over (PARTITION BY person_id) as max_steps,
+                               step_1_conversion_time
+        FROM
+          (SELECT *,
+                  if(latest_0 < latest_1
+                     AND latest_1 <= latest_0 + INTERVAL 14 DAY, 2, 1) AS steps ,
+                  if(isNotNull(latest_1)
+                     AND latest_1 <= latest_0 + INTERVAL 14 DAY, dateDiff('second', toDateTime(latest_0), toDateTime(latest_1)), NULL) step_1_conversion_time
+           FROM
+             (SELECT person_id,
+                     timestamp,
+                     step_0,
+                     latest_0,
+                     step_1,
+                     min(latest_1) over (PARTITION by person_id
+                                         ORDER BY timestamp DESC ROWS BETWEEN UNBOUNDED PRECEDING AND 0 PRECEDING) latest_1
+              FROM
+                (SELECT person_id,
+                        timestamp,
+                        if(event = 'user signed up', 1, 0) as step_0,
+                        if(step_0 = 1, timestamp, null) as latest_0,
+                        if(event = 'paid', 1, 0) as step_1,
+                        if(step_1 = 1, timestamp, null) as latest_1
+                 FROM
+                   (SELECT e.event as event,
+                           e.team_id as team_id,
+                           e.distinct_id as distinct_id,
+                           e.timestamp as timestamp,
+                           trim(BOTH '"'
+                                FROM JSONExtractRaw(e.properties, '$group_0')) as person_id
+                    FROM events e
+                    INNER JOIN
+                      (SELECT distinct_id,
+                              argMax(person_id, _timestamp) as person_id
+                       FROM
+                         (SELECT distinct_id,
+                                 person_id,
+                                 max(_timestamp) as _timestamp
+                          FROM person_distinct_id
+                          WHERE team_id = 2
+                          GROUP BY person_id,
+                                   distinct_id,
+                                   team_id
+                          HAVING max(is_deleted) = 0)
+                       GROUP BY distinct_id) AS pdi ON events.distinct_id = pdi.distinct_id
+                    WHERE team_id = 2
```

Comment:
So this query is not filtering by group_N being not blank. Missing `team` argument to Filter causing `simplify` not to be called or something else?